docs/builtins/Repl/env-sigs.md

```diff
@@ -24,21 +24,40 @@ The `env-sigs` function returns a string indicating that the transaction signatu
 
 ### Examples
 
-The following example demonstrates how to use the `env-sigs` function to set two transaction signature keys—"my-key" and "admin-key"—and capabilities:
+The following example illustrates using the `env-sigs` function to sign a transaction with a specific public key and capability in the Pact REPL:
+
+```pact
+(env-sigs [{"key": "58705e8699678bd15bbda2cf40fa236694895db614aafc82cf1c06c014ca963c", "caps": [(free.payments.PAYADMIN)]}])
+"Setting transaction signatures/caps"
+```
+
+The following example demonstrates how to use the `env-sigs` function to set transaction signatures and capabilities for two keys—"my-test-key" and "admin-key"—without using any public keys in the Pact REPL:
 
 ```pact
 (env-sigs [
-    {'key: "my-key", 'caps: [(accounts.USER_GUARD "my-account")]
+    {"key": "my-test-key", "caps": [(accounts.USER_GUARD "my-account")]
     }, 
-    {'key: "admin-key", 'caps: []}
-    ])
+    {"key": "admin-key", "caps": []}]
+)
 ```
 
-The following example illustrates using the `env-sigs` function to grant "any" key the MINT capability:
+The following example illustrates using the `env-sigs` function to grant "any" key the `MINT` capability for the `mint` function in a simplified transaction:
```

Comment:
I think "any" key is actually pretty bad messaging. It makes you think that we're using any key here. I would change `any` to `e4c6807d79d8bf4695e10e5678ebf72862f59b71f971d39dd3349f4beeacd6e3` or something like this, just to be clear.